Logo/bin/generate-assets.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+wd=`pwd`
+destdir="autogenerated"
+pngwidths="300 600 1200"
+totalassets=6
+
+find . -type f | grep svg >/dev/null || {
+	echo "No suitable files found. Sure we are in the right dir?"
+	return
+}
+
+# Dirty check for opened vectors
+grep -l -E ' d="[^"]*[^zZ]"' *.svg >"$destdir/.temp" &&  (
+	echo "This/those files seems to have open paths!";
+	cat  "$destdir/.temp"
+	echo
+)
+	
+# Loop through SVG files 
+count=0;
+titleerrors="";
+
+for file in *.svg ; do
+	let "count++"
+	asset="${file%.*}"
+	
+	# Check for valid SVG title
+	svgtitle=$(awk 'BEGIN{IGNORECASE=1;FS="<title>|</title>";RS=EOF} {print $2}' $file)
```

Comment:
Let's use `grep -Po` instead of `awk`, this will ensure better compatibility for OS X.

Reply:
Done... don't like awk much!